package.json

```diff
@@ -7,12 +7,17 @@
     "url": "git+https://github.com/edx/brand-edx.org.git"
   },
   "author": "edX",
-  "license": "UNLICENSED",
+  "license": "GPL-3.0-or-later",
   "bugs": {
     "url": "https://github.com/edx/brand-edx.org/issues"
   },
   "homepage": "https://github.com/edx/brand-edx.org#readme",
   "publishConfig": {
     "access": "public"
+  },
+  "devDependencies": {
+    "@edx/paragon": "^21.0.0-alpha.1",
```

Comment:
[clarification] Imports `@edx/paragon` to be able to source its core/default design token JSON files with `style-dictionary` in this repo. That said, the `v21.0.0-alpha.1` release of Paragon currently doesn't include the source `tokens` directory, so there is a manual step to copy them into `node_modules` as described in the README instructions.